frontend/src/pages/pipelines/global/runs/GlobalPipelineRuns.tsx

```diff
@@ -9,18 +9,20 @@ import PipelineRunVersionsContextProvider from '~/pages/pipelines/global/runs/Pi
 import GlobalPipelineRunsTabs from './GlobalPipelineRunsTabs';
 
 const GlobalPipelineRuns: React.FC = () => (
-  <PipelineCoreApplicationPage
-    title={pipelineRunsPageTitle}
-    description={pipelineRunsPageDescription}
-    getRedirectPath={(namespace) => `/pipelineRuns/${namespace}`}
-    overrideChildPadding
-  >
-    <EnsureAPIAvailability>
-      <PipelineRunVersionsContextProvider>
-        <GlobalPipelineRunsTabs />
-      </PipelineRunVersionsContextProvider>
-    </EnsureAPIAvailability>
-  </PipelineCoreApplicationPage>
+  <div data-testid="pipeline-runs-global-page">
```

Comment:
Can this `data-testid` be added directly to `PipelineCoreApplicationPage`? In that way we don't need to create an extra div container for it.

Reply:
I played around with this idea as well and figured I'd go with the least invasive path of just wrapping this JSX. I can switch it up though, it will just require a prop added to `ApplicationPage`.